bip-0002.mediawiki

```diff
@@ -190,7 +190,7 @@ The BIP editor may also change the status to Deferred when no progress is being
 
 A BIP may only change status from Draft (or Rejected) to Proposed, when the author deems it is complete, has a working implementation (where applicable), and has community plans to progress it to the Final status.
 
-BIPs should be changed from Draft or Proposed status, to Rejected status, upon request by any person, if they have not made progress in three years. Such a BIP may be changed to Draft status if the champion provides revisions that meaningfully address public criticism of the proposal, or to Proposed status if it meets the criteria required as described in the previous paragraph.
+BIPs should be changed from Draft or Proposed status, to Inactive status, upon request by any person, if they have not made progress in three years. Such a BIP may be changed to Draft status if any progress is made, or to Proposed status if it meets the criteria required as described in the previous paragraph.
```

Comment:
s/any progress is made/if any substantial revisions are made

I don't think editing a typo should be enough to get out of Inactive status. It has to be somewhat substantial. If not a significant update on the original proposal then at least an update on where it sits 3 years on with relation to what has been implemented on Core/other related proposals during that period

Reply:
See, a BIP may be inactive because there's no one working on a reference implementation or the like. In those cases, if the author or somebody makes progress on a reference implementation, they may not actually modify the BIP *at all* and still consider it "progress" enough that removing the inactive status is warranted.